### PR TITLE
Add 7002 ns support

### PR DESCRIPTION
**Pull request by krish2718**

### CODE REVIEW

**Review by sachinthegreen — APPROVED** [review]

**Review by rado17 — APPROVED** [review]

File: src/crypto/crypto_mbedtls-ec.c [hunk 1]
```diff
@@ -294,27 +297,29 @@ int crypto_ec_point_add(
 	return ret ? -1 : 0;
 }
 
+static int tls_ctr_drbg_random(void *ctx, unsigned char *buf, size_t len)
```
Comment (joerchan) [inline src/crypto/crypto_mbedtls-ec.c:300]: Why is it is required to feed random into mbedtls in this way?
We should have the required integration to get entropy alread.

Comment (frkv) [reply src/crypto/crypto_mbedtls-ec.c:300]: This is copying methodology in sockets_tls APIs in Zephyr. It is the same thing. TLS/DTLS  and legacy Mbed TLS APIs have frng/prng pointers they make use of.

**Review by frkv — APPROVED** [review]
LGTM

File: src/crypto/crypto_mbedtls-ec.c (hunk 1 above)
Comment (joerchan) [reply src/crypto/crypto_mbedtls-ec.c:300]: Ok, I can see that.

csrand may not be ctr_drbg. and sys_rand_get is not cryptographicaly secure.

I don't know if this is acceptable.

**Review by joerchan — APPROVED** [review]

File: src/crypto/crypto_mbedtls-ec.c (hunk 1 above)
Comment (frkv) [reply src/crypto/crypto_mbedtls-ec.c:300]: This is a copy-paste of the same function that sockets_tls.c is using.

**Pull request merged**